pkg/config/worker/config.go

```diff
@@ -19,34 +19,31 @@ type Config struct {
 	Encoder     encoder.Encoder
 	Emulator    emulator.Emulator
 	Environment shared.Environment
-	Worker      struct {
-		Monitoring monitoring.ServerMonitoringConfig
-		Network    struct {
-			CoordinatorAddress string
-			Zone               string
-		}
-		Server shared.Server
+	Worker      Worker
+	Webrtc      webrtcConfig.Webrtc
+}
+
+type Worker struct {
+	Monitoring monitoring.Config
+	Network    struct {
+		CoordinatorAddress string
+		Endpoint           string
+		PingEndpoint       string
+		Secure             bool
+		Zone               string
 	}
-	Webrtc webrtcConfig.Webrtc
-	Loaded bool
+	Server shared.Server
 }
 
 // allows custom config path
 var configPath string
 
 func NewConfig() (conf Config) {
-	if err := config.LoadConfig(&conf, configPath); err == nil {
-		conf.Loaded = true
-	}
+	_ = config.LoadConfig(&conf, configPath)
```

Comment:
I removed the worker's no-config option when it would pause initialization and request config from the coordinator because this stuff won't be useful.